Backends/CLX/input.lisp

```diff
@@ -141,11 +141,10 @@
       ((:button-press :button-release)
        (let ((modifier-state (clim-xcommon:x-event-state-modifiers *clx-port* state))
              (button (decode-x-button-code code)))
-         (if (and (eq event-key :button-press)
-                  (member button '(#.+pointer-wheel-up+
-                                   #.+pointer-wheel-down+
-                                   #.+pointer-wheel-left+
-                                   #.+pointer-wheel-right+)))
+         (if (member button '(#.+pointer-wheel-up+
```

Comment:
If you look at 14fd2c21311ae13d3753df76b51b597c724d1e6a, the intention seems to have been to generate *no* event for `:button-release` of a scroll wheel "button". Could you explain whether that idea was wrong? Your change would lead to a `pointer-scroll-event` for `:button-release`. Is that the intention?

Reply:
previous version generated release event for scrolling (because it was not pressed), so we have queued a spurious event anyway. without fix in push-button gadget in the other place that lead to activating the button callback when we were scrolling. if at all it should be:
```
(cond ((member ...)
            (when press
            (queue scroll))
       (press (queue press))
       (release (queue release)))
```

whether the idea itself was wrong I have no opinion.

Reply:
fixed in a way that we do not distribute event if it is a release button for one of the scrollings (so I've preserved the intent after all)